bigbluebutton-html5/private/locales/bg_BG.json

```diff
@@ -1,97 +1,9 @@
 {
-  "app.home.greeting": "Добре дошли, {0}! Вашата презентация ще започне всеки момент",
-  "app.userlist.usersTitle": "Потребители",
-  "app.userlist.participantsTitle": "Участници",
-  "app.userlist.messagesTitle": "Съобщения",
-  "app.userlist.presenter": "Лектор",
-  "app.userlist.you": "Вие",
-  "app.chat.submitLabel": "Изпрати",
-  "app.chat.inputLabel": "Въведи съобщение за {0}",
-  "app.chat.inputPlaceholder": "Съобщение {0}",
-  "app.chat.titlePublic": "Общ чат",
```

Comment:
Don't we need the first 11 entries? It looks like they are translated.
@antobinary we need your expert opinion.